### PR TITLE
Complete math magician setup

### PR DESCRIPTION
**Pull request by coderacheal**
### **_This pull request covers the following issues_**

- **A website consisting of 3 pages: Home, Calculator, and Quote.**
- **Every page, has its own Route.**
- **Used React Router to make the routes work.**

### CODE REVIEW

File: src/Components/Calculator.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { useState } from 'react';
 import calculate from '../logic/calculate';
+import styles from '../styles/Calculator.module.css';
 
 const Calculator = () => {
   const [state, setState] = useState({
```
Comment (KayLemba) [inline src/Components/Calculator.js]: While trying to run your app it was complied with the following errors:

Screenshot 2023-03-03 at 12 45 01 PM

This error is caused due to the importation path you used for you your styles. your styling files are saved in small letters but when you were importing them to your Component files you started with a capital letter

use this example to import your styling in all your component files.

import styles from '../styles/calculator.module.css';
import styles from '../styles/header.module.css';
import styles from '../styles/home.module.css';

Please lets pay attention to detail follow the above example and your app will work just fine.

File: src/Components/Quotes.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import styles from '../styles/quotes.module.css';
+// import Header from './Header';
+
+const Quotes = () => (
+  <div>
+    {/* <Header /> */}
```
Comment (KayLemba) [inline src/Components/Quotes.js]: Please kindly remove commented code if its not being used in this file for it makes your code base look untidy.

**Review by KayLemba — CHANGES_REQUESTED** [review]
Hi @coderacheal    👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️

Required Changes ♻️
Check the comments under the review in the inline comments.
 
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by amejid — APPROVED** [review]
Hi @coderacheal, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Navigation with React router successfully set up :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**